README.md

```diff
@@ -1,33 +1,50 @@
 # NPI_FILES
+
 repo to automate NPI file download, extract and load into a database and create an api for endpoint
 
-## Podman setup in script  
+## Podman setup in script
 
 work in progress... `pod_setup.sh`
 
-## Local Development for Django  
+## Local Development for Django
+
+Need to set up Postgres, either locally or in a container. The container is my choice as it makes it easier when testing with the containerized django app. The instructions below are specific to local Postgres environment.
```

Comment:
Let's add something that emphasizes the postgres db, this is gonna be essential for anyone running containers and or not having a local instance already running. I want to move away from the local one because that will make testing the container communication more difficult.